src/cr/cube/matrix.py

```diff
@@ -139,6 +139,33 @@ def table_margin_unpruned(self):
         return self._base_matrix.table_margin
 
 
+class StripeWithHidden(object):
+    """Stripe with hidden rows removed.
+
+    A row can be hidden explicitly by the user, or it can be automatically hidden when
```

Comment:
I'd add some description here, that the `row` is actually _just_ a single element inside the row, in this particular case.